### PR TITLE
Build hostplumber image via github action

### PR DESCRIPTION
**Pull request by shweta50**
Image built via TC fails on certain hardware with core dumped error. 
Building image via github action with runs on : ubuntu-latest. 

I have validated the image on the hardware it was failing on. 

JIRA: https://platform9.atlassian.net/browse/PMK-6478

### CODE REVIEW

File: .github/workflows/hostplumber-multiarch-build-and-push.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      BUILD_ARGS: ""
+      PLATFORMS: linux/amd64
+      BRANCH: ${{ github.head_ref || github.ref_name }}
+      PUSH: true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        run: 
+          echo "TAG=$(echo "${{ env.BRANCH }}" | tr -d /)" >> $GITHUB_ENV
```
Comment (cruizen) [inline .github/workflows/hostplumber-multiarch-build-and-push.yml:25]: In the existing Makefile, we don't simply use the git branch as tag. We call the `get-label.sh` script [here](https://github.com/platform9/luigi/blob/master/hostplumber/Makefile#L3) from `Makefile`. 
The script uses the tag if the current git revision is tagged. Else the branch name. And then applies a suffix (either the Teamcity build ID if there is one or  the git revision SHA ID)

**Review by cruizen — COMMENTED** [review]
Can we call the make targets `img-test` / `test` to execute tests too? 
Also see my comment on building the image tag.

File: .github/workflows/hostplumber-multiarch-build-and-push.yml (hunk 1 above)
Comment (shweta50) [reply .github/workflows/hostplumber-multiarch-build-and-push.yml:25]: What if we use ${{ github.run_number }} here instead.
Tag will be `branchnme-${{ github.run_number }}`

Comment (cruizen) [reply .github/workflows/hostplumber-multiarch-build-and-push.yml:25]: Sure, the `run_number` is a good replacement for `TEAMCITY_BUILD_ID`.

Comment (cruizen) [reply .github/workflows/hostplumber-multiarch-build-and-push.yml:25]: We can also modify the Makefile to use the run_number as an environment parameter and call the make target from Github action?

Comment (shweta50) [reply .github/workflows/hostplumber-multiarch-build-and-push.yml]: Updated to call make target.

**Review by cruizen — APPROVED** [review]
Thanks. Please increment the version of Luigi and hostplumber to v0.5.7

**Pull request merged**